### PR TITLE
RAINCATCH-1318 updates to master.adoc

### PR DESCRIPTION
**Pull request by austincunningham**
## Motivation:
reorganise all documentation and include in master.adoc  for publication

## What
Added in changes to the ui documentation and integrations  documentation

- [x] Documentation

### CODE REVIEW

File: docs/workforce-management-framework/topics/con-wfm-intergrations.adoc [hunk 1]
```diff
@@ -1,26 +1,26 @@
-= {Feature} intergration
+//= {PROJECT_NAME} intergration
 
-An intergration is the use of existing opensource projects to solve specific needs for {Feature}. 
-{Feature} has a number of intergrations that leverage other opensource projects for there expertises in paticular fields.
-This enabled the inclusion of domain expertises in the areas of security and sync in the developement of {Feature}.
+An intergration is the use of existing opensource projects to solve specific needs for {PROJECT_NAME}.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/con-wfm-intergrations.adoc]: Why we doing that?

File: docs/workforce-management-framework/topics/ref-demo-app.adoc [hunk 2]
```diff
@@ -1,13 +1,13 @@
 [id='{context}-ref-demo-app']
-= {PROJECT_NAME} Demo Application Reference Material
+= {PROJECT_NAME} Application Reference Material
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/ref-demo-app.adoc:2]: ❤️

File: docs/workforce-management-framework/topics/con-wfm-intergrations.adoc [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
 
-An intergration is the use of existing opensource projects to solve specific needs for {Feature}. 
-{Feature} has a number of intergrations that leverage other opensource projects for there expertises in paticular fields.
-This enabled the inclusion of domain expertises in the areas of security and sync in the developement of {Feature}.
+An intergration is the use of existing opensource projects to solve specific needs for {PROJECT_NAME}. 
+{PROJECT_NAME} has a number of intergrations that leverage other opensource projects for there expertises in paticular fields.
+This enabled the inclusion of domain expertises in the areas of security and sync in the developement of {PROJECT_NAME}.
 
-== Security intergrations
+//== Security intergrations
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/con-wfm-intergrations.adoc]: Why we commenting this? Just want to understand? 
Do you want to hide this for the moment? If yes it seems reasonable.

File: docs/workforce-management-framework/upstream-1/master.adoc [hunk 4]
```diff
@@ -86,36 +98,6 @@ include::topics/con-logging.adoc[leveloffset=+1]
 //include::defining-raincatcher-security.adoc[leveloffset=+2]
 
 
-
```
Comment (wtrocki) [inline docs/workforce-management-framework/upstream-1/master.adoc:89]: Why?

File: docs/workforce-management-framework/topics/con-wfm-intergrations.adoc (hunk 3 above)
Comment (austincunningham) [reply docs/workforce-management-framework/topics/con-wfm-intergrations.adoc]: The same header in master.adoc so seeing it twice

Comment (wtrocki) [reply docs/workforce-management-framework/topics/con-wfm-intergrations.adoc]: Why not remove this completely then. 
I'm just seeing most of this for the first time and I think we need to minimize confusion by reducing number of files and references. Docs are small but they overloaded by references.

File: docs/workforce-management-framework/upstream-1/master.adoc (hunk 4 above)
Comment (austincunningham) [reply docs/workforce-management-framework/upstream-1/master.adoc:89]: old content no longer needed

**Comment by wtrocki** [comment]
Is this PR removing any content from the docs?
if not feel free to merge :smile:

File: docs/workforce-management-framework/topics/con-wfm-intergrations.adoc (hunk 3 above)
Comment (austincunningham) [reply docs/workforce-management-framework/topics/con-wfm-intergrations.adoc]: agree will remove the content as it adds nothing , and just clutters up the toc when rendered.

**Pull request merged**